package.json

```diff
@@ -145,6 +145,7 @@
       "libs/*",
       "packages/*",
       "enterprise/packages/*",
+      "packages/notification-center-angular/projects/*",
```

Comment:
Necessary to have the `lerna publish` command bump the versions for deeply nested angular projects.